### PR TITLE
Prevent hover/click on disabled .close links

### PR DESCRIPTION
**Pull request by ysds**
The `.close` is supported with ` <a> `, but it is still hoverable/clickable when it is disabled with the` .disabled` class. This PR prevents hover/click on disabled .close links.

Demo:
* Before: https://codepen.io/anon/pen/NOVeeN
* After: https://codepen.io/anon/pen/EdzGOE

### CODE REVIEW

**Comment by MartijnCuppens** [comment]
Unfortunately, `pointer-events` is not supported in IE10

**Comment by ysds** [comment]
yes, the same as https://getbootstrap.com/docs/4.1/components/buttons/#disabled-state

**Comment by ysds** [comment]
I’ll add commit later to prevent visual changes on IE10.

**Review by mdo — APPROVED** [review]

**Comment by XhmikosR** [comment]
@MartijnCuppens: LGTY?

File: scss/_close.scss [hunk 1]
```diff
@@ -7,11 +7,15 @@
   text-shadow: $close-text-shadow;
   opacity: .5;
 
+  // Override <a>'s hover style
+  &:hover {
```
Comment (MartijnCuppens) [inline scss/_close.scss]: We should use the hover mixin here:
```suggestion
  @include hover {
```

**Review by MartijnCuppens — DISMISSED** [review]

**Comment by XhmikosR** [comment]
I still can't merge until the GitHub situation is over :P

**Comment by XhmikosR** [comment]
@ysds: can you fetch upstream and rebase?

**Comment by XhmikosR** [comment]
@ysds: this is not a rebase.

**Comment by ysds** [comment]
@XhmikosR yes.. I made some mistake. What should I do?

**Comment by XhmikosR** [comment]
Let me fix it for you. You can read a bit about interactive git rebase for the future :)

**Comment by ysds** [comment]
Thanks :)

**Pull request merged**